src/iap/shared/pipes/is-control-invalid.pipe.ts

```diff
@@ -0,0 +1,18 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+
+@Pipe({
+	name: 'isControlInvalidErrors',
+	pure: false,
+})
+export class IsControlInvalidPipe implements PipeTransform {
+	transform(formGroup: FormGroup, controlName: string) {
+		const control = formGroup.controls?.[controlName];
+
+		if (!control) {
+			return null;
+		}
+
+		return control.invalid && (control.touched || control.dirty);
+	}
```

Comment:
You can just work with `AbstractControl` entity:
```suggestion
	transform(control: AbstractControl) {
		return control.invalid && (control.touched || control.dirty);
	}
```